app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="ai.elimu.appstore">
```

Comment:
@jo-elimu We can keep the package here and dismissing all the package prefix at the start of every following component in this manifest file.

Reply:
@tuancoltech Yes, I agree with you that that would be a better (more maintainable) solution. Just curious: Why does the Android Studio upgrade assistant encourage removing `package` from the manifest file? They should probably be aware that removing it causes the package prefix problem?

Reply:
From why I can see at https://developer.android.com/guide/topics/manifest/manifest-element#package, it hasn't been deprecated. So then it's probably best to just leave it in the manifest file. I'll revert the changes.

Reply:
> @tuancoltech Yes, I agree with you that that would be a better (more maintainable) solution. Just curious: Why does the Android Studio upgrade assistant encourage removing `package` from the manifest file? They should probably be aware that removing it causes the package prefix problem?

@jo-elimu Probably the upgrade assistant issue.

Reply:
Resolved: https://github.com/elimu-ai/appstore/pull/257/commits/f1e4b499fbba0332a19f5850517743e2a91610ee